src/pages/Scholarships.jsx

```diff
@@ -41,6 +43,9 @@ function ScholarshipsPage() {
         (loading && <CircularProgress className={classes.progress} />) || (
           <ScholarshipList scholarships={scholarships} />
         )}
+      <Divider />
```

Comment:
Instead of a divider how about adding margins above and below the Pagination component so that it has space above/below it? 

This'll look more like the Figma design.

Relevant info: [styles](https://material-ui.com/styles/basics/). I suggest a bottom/top margin of 24px

Reply:
I think the spacing issue here is still relevant. The only thing that's changing is the Pagination component.